src/base/_defaults.scss

```diff
@@ -53,15 +51,15 @@ ul {
 
 body {
   background-color: #fff; /* 1 */
-  color: colors.$text-dark; /* 2 */
+  color: tokens.$color-text-dark; /* 2 */
```

Comment:
My least favorite thing about Style Dictionary in a recent client project is how every single token group is housed under a `tokens.` object. If there isn't a way to recreate this with Style Dictionary, I'm honestly tempted to say this transition isn't worth it right now. (That may seem dramatic, but it's really a pain to type dozens of times!)

Reply:
That makes sense, and it's absolutely something we can address by generating some useful subsets of tokens. I'll talk to you about what subsets would be most useful.